plugins/optimization-detective/tests/test-class-od-html-tag-processor.php

```diff
@@ -426,6 +426,7 @@ public function test_html_tag_processor_wrapper_methods(): void {
 			$open_tag = $processor->get_tag();
 			if ( 'HTML' === $open_tag ) {
 				$processor->set_attribute( 'lang', 'es' );
+				$processor->set_attribute( 'class', 'foo' ); // Unchanged.
```

Comment:
What does this comment mean?

Reply:
This is just to demonstrate that calling `set_attribute()` and passing a value that is the same as is currently set will not cause the `data-od-replaced-class` metadata attribute to be added. To make this more clear:

```suggestion
				$processor->set_attribute( 'class', 'foo' ); // Unchanged from source to test that data-od-replaced-class metadata attribute won't be added.
```